ietf-layer0-types-tree.txt

```diff
@@ -329,18 +329,17 @@ module: ietf-layer0-types
     +--ro rx-channel-power-max?     power-dbm
     +--ro rx-total-power-max?       power-dbm
   grouping common-transceiver-configured-param:
-    +--ro line-coding-bitrate?   identityref
+    +-- line-coding-bitrate?   identityref
     +-- tx-channel-power?      power-dbm-or-null
     +--ro rx-channel-power?      power-dbm-or-null
```

Comment:
common-transceiver-configured-param should contain only configurable attributes. Do we need to split this grouping into two - one for r/w the other for r/o attributes?